editor/src/templates/property-controls-info.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- ogTags -->
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="%UTOPIA_DOMAIN%/editor/preview.css?hash=%UTOPIA_SHA%"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="%UTOPIA_DOMAIN%/editor/draft.css?hash=%UTOPIA_SHA%"
+      crossorigin="anonymous"
+    />
+    <!-- Begin Generated Utopia External Links -->
+    <!-- End Generated Utopia External Links -->
+  </head>
+  <body>
+    <!-- projectIDScript -->
+    <div id="preview"></div>
+    <div id="preview-ui"></div>
+  </body>
```

Comment:
Seems pretty much all of this can go in the bin?

Reply:
Very true.

Reply:
Well, sorry, we'd need the `<head>` tags, but no contents. I'm pretty sure we don't need the `<body>` tags.

Reply:
Snipped in 3f385e6b8.